docs/csharp/index.md

```diff
@@ -4,12 +4,37 @@ description: Learn how the C# Guide can help you gain extensive knowledge on C#,
 keywords: .NET, .NET Core
 author: BillWagner
 
-ms.date: 08/03/2016
+ms.date: 08/23/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: devlang-csharp
 ms.devlang: csharp
 ms.assetid: 52db8280-0e53-40cf-858b-e8eef3997dea
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "documentation [C#]"
+  - "Help, Visual C# documentation"
+  - "C# language"
+  - "Visual C#"
+caps.latest.revision: 47
```

Comment:
this is not from caps, so don't add this

Reply:
I wasn't sure, because I did incorporate some of that content.